Olé shows nag popups now, seems to be using Clarin's.

ramiro opened the pull request.

Commit:
Olé shows nag popups now, seems to be using Clarin's.

MissFilly commented:
This is great! Thanks for the PR! Someone actually suggested this to me but I failed to reproduce the "problem" in Olé. Did you run the modified add-on locally and confirm that this code removes the popups?

ramiro commented:
I saw the popup on May 30 so I implemented:
- The `.modalLoginPase` blocking part via  a custom _uBlock Origin_ rule.
- The `:root {overflow-y: visible ...` part via a `userContent.css` file (a very brute force one, I know.)

because I wasn't sure Firefox would allow me to use a modified addon .zip including these changes and I don't have the Firefox addons dev enviroment to properly pack a modified one. After that the popups disappeared and I was happy, and I opened the PR.

But the problem is that since your comment above I've removed both of them ... and I still don't see the popups.

So maybe they've removed them, or I caught a test deployment?

ramiro commented:
I had left the rules desccribed above inactive and today I started to see the popup consistently on every article. I activated them again and this made the popup dissappear.

So I can say these two rules do actually work.
